atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -265,6 +279,27 @@ public String toString() {
         return pool;
     }
 
+    private void logThreadStates() {
+        ThreadMXBean threadBean = ManagementFactory.getThreadMXBean();
+        for (ThreadInfo info : threadBean.getThreadInfo(threadBean.getAllThreadIds())) {
+            // we're fairly good about annotating our C* pool thread names with the current action
+            if (log.isTraceEnabled()) {
+                log.trace("active thread",
+                        UnsafeArg.of("threadName", info.getThreadName()),
```

Comment:
This is fine for now. I think internal log frameworks usually report this as safe (separately as part of the thread field)

Reply:
That's a bit bad, since the thread name here I think could be something like "connected to cassandra node {unsafe hostname or IP} doing getRows call on table {unsafe tablename}"